contractsV5/FeeHandler.sol

```diff
@@ -0,0 +1,200 @@
+pragma solidity 0.5.11;
+
+import "./IKyberDAO.sol";
+import "./IFeeHandler.sol";
+import "./PermissionGroupsV5.sol";
+import "./IKyberNetworkProxy.sol";
+import "./UtilsV5.sol";
+import "./IERC20.sol";
+import "./IBurnableToken.sol";
+
+contract FeeHandler is IFeeHandler, Utils {
+
+    IKyberDAO public kyberDAOContract;
+    // if I use the interface, I am not able to get the KyberNetwork internal contract from the proxy..
+    IKyberNetworkProxy public kyberNetworkProxyContract;
+    address public kyberNetworkContract;
+    address public knc;
+    uint public burnBlockInterval;
+    uint public lastBurnBlock;
+    uint constant ETH_TO_BURN = 10**19;
```

Comment:
please place constants together and dynamic together.
I think the convention is constant first. but not sure.

Reply:
noted and updated